internal/runner/executer.go

```diff
@@ -23,21 +23,28 @@ func (r *Runner) ExecuteCrawling() error {
 
 	wg := sizedwaitgroup.New(r.options.Parallelism)
 	for _, input := range inputs {
+		if input == "" {
```

Comment:
This check can be done at https://github.com/projectdiscovery/katana/blob/main/internal/runner/options.go#L89